library/src/lib/date-picker/date-picker.component.ts

```diff
@@ -132,7 +132,9 @@ export class DatePickerComponent implements OnInit, OnDestroy, ControlValueAcces
     }
 
     getInputValue(e) {
-        this.dateFromDatePicker.next(e);
+        if (e !== this.inputFieldDate) {
```

Comment:
does this work? usually I compare dates with getTime() but if this works then good

Reply:
Yes it works